projects/packages/assets/src/class-assets.php

```diff
@@ -323,9 +338,12 @@ public static function normalize_path( $path ) {
 	 *  - `nonmin_path`:      (string) Non-minified script path.
 	 *  - `textdomain`:       (string) Text domain for the script. Required if the script depends on wp-i18n.
 	 *  - `version`:          (string) Override the version from the `asset_path` file.
+	 *  - `strategy`:         (string) Specify a script strategy to use, eg. `defer` or `async`. Default is `""`.
```

Comment:
The rest are in alphabetical order, let's keep that going.
```suggestion
	 *  - `strategy`:         (string) Specify a script strategy to use, eg. `defer` or `async`. Default is `""`.
	 *  - `textdomain`:       (string) Text domain for the script. Required if the script depends on wp-i18n.
	 *  - `version`:          (string) Override the version from the `asset_path` file.
```